src/lib/qnio/nio_server.c

```diff
@@ -452,14 +452,16 @@ qns_server_start(char *node, char *port)
                             {
                                 nioDbg("SSL_error is want read or want write %d", ret);
                                 /* put fd in epoll loop to try again */
-                                event.events = EPOLLIN;
+                                event.events = EPOLLIN | EPOLLOUT;
```

Comment:
Checked in a new patch which sets EPOLLIN for WANT_READ and EPOLLOUT for WANT_WRITE. Surprisingly I am always getting a EPOLLIN for the fd on incomplete ssl accept. I never see an EPOLLOUT.